docs/DeveloperGuide.md

```diff
@@ -575,7 +575,33 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
   * Steps 1a1-1a2 are repeated until the index entered is valid.
   * Use case resumes from step 2.
 
-**Use case: UC4 - Find an employee**
+**Use case: UC4 - Edit an employee**
+
+**MSS**
+
+1. User requests to edit an employee with index and new details.
+2. EmployeeManager edits the employee.
+3. EmployeeManager shows an updated list of employees.
+4. Use case ends.
+
+**Extensions**
+
+* 1a. The given index is invalid.
+    * 1a1. EmployeeManager informs user that the given index is invalid.
+    * 1a2. User enters new index.
+    * Steps 1a1-1a2 are repeated until the index entered is valid.
+    * Use case resumes from step 2.
+* 1b. EmployeeManager detects invalid information.
+    * 1b1. EmployeeManager informs user that some information is invalid.
+    * 1b2. EmployeeManager requests user to enter details of the employee.
+    * 1b3. User enters the requested details.
+    * Steps 1b1 - 1b3 are repeated until the data entered are correct.
```

Comment:
Similarly, we discussed changing "are correct" to "is correct" for the `add` use case. Perhaps to make it better, we can write "Steps 1b1 - 1b2 are repeated until the information entered is valid"?